src/tag/tag.js

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import React from 'react';
+import {getOverride, getOverrideProps} from '../helpers/overrides';
+import type {PropsT, DefaultPropsT, StatelessStateT} from './types';
+import {Action as StyledAction, Root as StyledRoot} from './styled-components';
+import {STYLE} from './constants';
+
+class StatelessTag extends React.Component<PropsT, StatelessStateT> {
+  static defaultProps: DefaultPropsT = {
+    overrides: {},
+    onActionClick: () => {},
+    color: STYLE.blue,
+    disabled: false,
+    isFocused: false,
+    isHovered: false,
+    isActive: false,
```

Comment:
How do all of these work? It looks like you're not specifying them in `<Select>`.

Should we remove these for now and rely on CSS pseudo selector like `:focus` and `:hover`?

Reply:
ok